services/web/server/src/simcore_service_webserver/application_settings_utils.py

```diff
@@ -259,6 +258,7 @@ def _set_if_disabled(field_name, section):
         envs["SMTP_HOST"] = section.get("host")
         envs["SMTP_PORT"] = section.get("port")
         envs["SMTP_TLS_ENABLED"] = section.get("tls")
```

Comment:
The changes you are doing here are inconsistent. 
The idea of this functions is to translate configs (i.e. the old and deprecated configuration method) into environment variables (envs in short). Then our pydantic settings, capture this envs and produce Settings. 

The main reason, is that there are a lot of tests that are still relying on old configs fixtures. Using this function we translate config->envs and then the new Settings can capture the expected fixture.

In ``login/settings.py`` you removed  ``SMTP_TLS_ENABLED`` and added a new field called ``SMTP_PROTOCOL``.  This entry was not in the old config so  ... i see some inconsistencies here:

1. you remove  reading ``SMTP_TLS_ENABLED`` envs into ``"tls"`` config but NOT the other way around (``envs["SMTP_TLS_ENABLED"] = section.get("tls")``)
2. you add a new envvars ``"SMTP_STARTTLS_ENABLED"`` from a NEW config ``"starttls"`` that i do not think exists. And more over, configs are deprecated so we are not allowed to add more

Reply:
I addressed these inconsistencies by removing the env var `SMTP_TLS_ENABLED ` from the code in all places. Thank you for the explanation :)